web_pwa_oca/tests/test_web_pwa_oca_controller.py

```diff
@@ -0,0 +1,141 @@
+            b"jYwIiBwcmVzZXJ2ZUFzcGVjdFJhdGlvPSJub25lIiBzdHlsZT0iaW1hZ2UtcmVuZG" \
+            b"VyaW5nOm9wdGltaXplUXVhbGl0eSIgaWQ9ImltYWdlMTAiIHg9IjAiIHk9IjAiLz4" \
+            b"KICA8cmVjdCB5PSIwIiB4PSIwIiBoZWlnaHQ9IjYwIiB3aWR0aD0iNjAiIGlkPSJy" \
+            b"ZWN0NDE3MSIgc3R5bGU9Im9wYWNpdHk6MTtmaWxsOiM0NzQ3OTc7ZmlsbC1vcGFja" \
+            b"XR5OjEiLz4KICA8cGF0aCBzdHlsZT0ib3BhY2l0eToxO2ZpbGw6IzNjM2M2NDtmaW" \
+            b"xsLW9wYWNpdHk6MSIgZD0iTSAwLDM2LjY3MzY4MiAxNS4xNDI3ODEsNy4yNjgyNzY" \
+            b"1IDI5LjQ5ODE4Niw3LjMyMjc0NjUgNDYuMzQ5MTUyLDkuNTc2MDExNSA0NS42MjAw" \
+            b"NDEsNDIuNTYyNTcgMzIuMzU4NTkyLDU5LjgyMDIzIDAuMDQzNjIsNjAgWiIgaWQ9I" \
+            b"nJlY3QzMzM4Ii8+CiAgPGcgdHJhbnNmb3JtPSJtYXRyaXgoMC4yMDAzNjAzMiwwLD" \
+            b"AsMC4yMDAzNjAzMiwtNDguNTUzODIxLC03Ni41ODk2OTkpIiBpZD0ibGF5ZXIxIiB" \
+            b"zdHlsZT0iZmlsbDojZmZmZmZmIj4KICAgIDxwYXRoIGlkPSJwYXRoMTMwIiBkPSJt" \
+            b"IDMyNC42Nzc3Myw0MTguMTI1OTQgYyAtNy43OTMsMCAtMTQuMTY4NjgsNi4zMzg5I" \
+            b"C0xNC4xNjg2OCwxNC4xNDYyMiAwLDcuODA3MzEgNi4zNzU2OCwxNC4xNDYzMyAxNC" \
+            b"4xNjg2OCwxNC4xNDYzMyBsIDEyLjE3NjU1LDAgYyA3Ljc5Mjk5LDAgMTQuMTIzODc" \
+            b"sLTYuMzM5MDIgMTQuMTIzODcsLTE0LjE0NjMzIDAsLTcuODA3MiAtNi4zMzA4OCwt" \
+            b"MTQuMTQ2MjIgLTE0LjEyMzg3LC0xNC4xNDYyMiBsIC0xMi4xNzY1NSwwIHogbSA2N" \
+            b"y42ODcyNiwwIGMgLTcuODIxNjUsMCAtMTQuMTY4Nyw2LjMzODkgLTE0LjE2ODcsMT" \
+            b"QuMTQ2MjIgbCAwLDEyMC45Mzc0IGMgMCw3LjgzNTg2IDYuMzQ3MDQsMTQuMTQ2MjI" \
+            b"gMTQuMTY4NywxNC4xNDYyMiA3LjgwNzI5LDAgMTQuMTQ2MzEsLTYuMzEwMjYgMTQu" \
+            b"MTQ2MzEsLTE0LjE0NjIyIGwgMCwtMTA2Ljc5MDk1IDUyLjk1ODk3LDAgYyA3LjgwN" \
+            b"zI5LDAgMTQuMTQ2MjEsLTYuMzM5MDIgMTQuMTQ2MjEsLTE0LjE0NjM0IDAsLTcuOD" \
+            b"A3MzEgLTYuMzM4OTIsLTE0LjE0NjIxIC0xNC4xNDYyMSwtMTQuMTQ2MjEgbCAtNjc" \
+            b"uMTA1MjgsMCB6IG0gLTY1LjExMzE2LDE2Mi4zMDE2NyBjIC0xLjgwMTIyLDAuMTgz" \
+            b"MzYgLTMuNTk1NjcsMC43MTM5NyAtNS4yODI1LDEuNjMzOTkgLTYuNzE4NTgsMy42O" \
+            b"TU4MiAtOS4xOTQxMiwxMi4yMDE1MiAtNS40ODM4NywxOC44OTE0NyAxNS4yNzA4M" \
+            b"SwyNy43NzY4MyA0NC4xMjAyMyw0NC45MDExMSA3NS41MjEzOCw0NC45MDExMSAwLj" \
+            b"Q3MjE3LDAgMC45NTEyLC0wLjAxMiAxLjQxMDE4LC0wLjA0NTggMzAuNzg1MTYsLTA" \
+            b"uNDU4NDEgNTkuMTUzODIsLTE3LjY5NTM1IDc0LjA2NjQ5LC00NC44NTYyMSAzLjcy" \
+            b"NDYsLTYuNzQ3MjMgMS4yNjMyNiwtMTUuMTk1NjMgLTUuNDgzOTcsLTE4Ljg5MTU3I" \
+            b"C02LjcwNDI3LC0zLjY4MTYyIC0xNS4xNjY4NywtMS4yMjg0MyAtMTguODkxNDcsNS" \
+            b"40NjE1MSAtMTAuMTI4MDEsMTguNDk0MDIgLTI5LjM0NjQzLDMwLjE3OTAyIC01MC4" \
+            b"xNjExNSwzMC41MDg1MSBsIC0wLjkzOTc1LDAgYyAtMjEuMjQ0NDgsMCAtNDAuNzc0" \
+            b"MzcsLTExLjYxMzI4IC01MS4xNDU5NCwtMzAuNTA4NTEgLTIuNzUwNDYsLTUuMDE3N" \
+            b"DMgLTguMjA1NTQsLTcuNjQ2MTggLTEzLjYwOTA3LC03LjA5NTUyIHoiIHN0eWxlPS" \
+            b"JmaWxsOiNmZmZmZmY7ZmlsbC1vcGFjaXR5OjE7ZmlsbC1ydWxlOm5vbnplcm87c3R" \
+            b"yb2tlOm5vbmUiLz4KICA8L2c+Cjwvc3ZnPgo="
```

Comment:
wouldn't it be more maintainable to drop a svg file somewhere and doing something similar to `b64encode(open(that_file, "b").read())`?